overlord/managers_test.go

```diff
@@ -184,27 +184,21 @@ apps:
 }
 
 const fooSearchHit = `{
-    "_embedded": {
-        "clickindex:package": [
-            {
-                "anon_download_url": "@URL@",
-                "architecture": [
-                    "all"
-                ],
-                "channel": "stable",
-                "content": "application",
-                "description": "this is a description",
-                "download_url": "@URL@",
-                "icon_url": "@ICON@",
-                "origin": "bar",
-                "package_name": "foo",
-                "revision": @REVISION@,
-                "snap_id": "idididididididididididididididid",
-                "summary": "Foo",
-                "version": "@VERSION@"
-            }
-        ]
-    }
+	"anon_download_url": "@URL@",
+	"architecture": [
+	    "all"
+	],
+	"channel": "stable",
+	"content": "application",
+	"description": "this is a description",
+	"download_url": "@URL@",
+	"icon_url": "@ICON@",
+	"origin": "bar",
+	"package_name": "foo",
+	"revision": @REVISION@,
+	"snap_id": "idididididididididididididididid",
```

Comment:
given that we are touching this, probably a good time to add developer_id here and checking it (if he are checking snap_id)